<?xml version="1.0"?>
<launch>
    <node pkg="pegasus_ros" type="pegasus_controller.py" name="pegasus_controller" respawn="false" output="screen">
        <rosparam param="agents">
            [
                ["uav0", ["localhost", 4444]],
            ]
        </rosparam>
        <param name="agents_hover_height" value="10"/>
        <param name="grid_size" value="10"/>
        <param name="map_origin_topic" value="/local_xy_origin"/>
    </node>
</launch>
